test/new_relic/gem_notifications_tests.rb

```diff
@@ -3,19 +3,18 @@
 # See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
 # frozen_string_literal: true
 
-# require_relative '../test_helper'
```

Comment:
Can this line be removed?

Reply:
I think so. We don't actually need that test_helper file in my tests

Reply:
Address with [7196d](https://github.com/newrelic/newrelic-ruby-agent/pull/1273/commits/7196d8c02d25b865deb2e3faf318b78d2fb43785)